TestRunners/Utils.py

```diff
@@ -0,0 +1,73 @@
+import os
+
+TERMINAL_TEXT_STYLES = {
+    "BOLD_ON": "\033[1m",
+    "BOLD_OFF": "\033[22m",
+    "RED_COLOR": "\033[31m",
+    "GREEN_COLOR": "\033[32m",
+    "RESET_COLOR": "\033[0m",
+}
+
+
+def split_string_by_n_characters(input_string: str, n: int) -> "list[str]":
```

Comment:
I suggest writing unit tests for each of the methods in this file.

Reply:
(and running them in CI)

Reply:
Considering how small each function is and how their importance on the codebase is relatively small, what usage is obtained by writing unit tests for them? I feel all the testing I did while developing the Python code made me pretty confident in the reliability of these functions

Reply:
> I feel all the testing I did while developing the Python code made me pretty confident in the reliability of these functions

As far as I'm concerned, this testing did not occur (because it is not automated, so I can't observe it). More importantly, going forward there is no way to guarantee that these functions will continue to work as expected without writing tests that cover them.

A good software engineering practice is to always write automated tests for any code that you write, check those tests in, and run them in CI, because manual testing is notoriously unreliable and doesn't preserve any record of the functions expected behavior.